keps/sig-auth/20190607-certificates-api.md

```diff
@@ -101,15 +100,22 @@ committing it to storage so that they can be used later during CSR approval. The
 information contained in the spec is immutable after the request is created.
 
 An approver updates approval status of the CertificateSigningRequest via the
-CertificateSigningRequestStatus. The approval condition can only be updated via
-the `/approval` subresource allowing approval permission to be authorized
+CertificateSigningRequestStatus. `Approved`/`Denied` conditions can only be set via
+the `/approval` subresource, allowing approval permission to be authorized
```

Comment:
Are we happy with this?  I think I may prefer setting status in a v1.

Reply:
Agree that the subpresource here is obsolete now that we have the admission control. We should definitely consider simplifying the strategy here for v1. I recommend that we revisit this separately once we are sketching the GA surface. Maybe record it here: https://github.com/kubernetes/kubernetes/issues/69836

Reply:
I considered that, but thought it was dangerous for subresource-level policies to [mean different things against different versions of the API](https://github.com/kubernetes/enhancements/pull/1691/files#r411658249).